image_understanding/image_understanding.py

```diff
@@ -0,0 +1,38 @@
+from openai import OpenAI
+import streamlit as st
+import base64
+import os
+
+model_service = os.getenv("MODEL_SERVICE_ENDPOINT",
+                          default="http://localhost:8001/v1")
```

Comment:
Why fallback on localhost? Assuming everything runs inside a container this `localhost` will never be reachable, right? How about failing if `MODEL_SERVICE_ENDPOINT` is empty?

Reply:
We could. However, I find this is fairly helpful while developing, before we've containerized anything.